data/transformers/geojson.py

```diff
@@ -33,10 +33,16 @@ def data_array_to_geojson(data_array: xr.DataArray, lat_key: str, lon_key: str)
     if data_array.ndim != 2:
         raise ValueError(f"Data is not a 2D field: {data_array.shape}")
 
+
     # Need to ensure that data values are 64-bit floats (i.e. Python builtin float) because
     # that's the only type of float that json will serialize without a custom serializer.
     # Floats from netCDF4 datasets are often 32-bit.
-    data = trunc(data_array.data.astype(float))
+    data = np.nan_to_num(
+        trunc(
+            data_array.astype(float)
+        ),
+        nan=np.Infinity
```

Comment:
nans can't be serialized into json natively so filter them out with a standard constant.